docker-app/qfieldcloud/subscription/models.py

```diff
@@ -524,10 +543,20 @@ class UpdateSubscriptionKwargs(TypedDict):
     @property
     @deprecated("Use `AbstractSubscription.active_storage_total_bytes` instead")
     def active_storage_total_mb(self) -> int:
+        if self.plan.is_seat_flexible:
+            return self.included_storage_mb + self.active_storage_package_mb
+
         return self.plan.storage_mb + self.active_storage_package_mb
```

Comment:
I guess we can either kill this method, or just `self.active_storage_total_bytes() / 1000 / 1000`